Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRSupport.cs

```diff
@@ -317,6 +318,7 @@ public static void Initialize()
 
             InputSystem.onFindLayoutForDevice += XRLayoutBuilder.OnFindLayoutForDevice;
 
+#if ENABLE_VR
```

Comment:
Why is this one ENABLE_VR, where all other checks are for UNITY_XR_AVAILABLE?

Reply:
removed with layouts